### ISSUE TITLE
Provide a utility to extract features from texts?

### ISSUE THREAD

**Issue by handsomezebra** [zihangdai/xlnet#26]
Congratulations on the wonderful work!

I can't wait to try the model. Do you have plan to provide a utility to extract vectors for some sample texts. Just as the extract_features.py in the BERT repo? Or is there other ways to easily run the model against some raw texts?

**Comment by kimiyoung** [comment]
Good point. [This example snippet](https://github.com/zihangdai/xlnet#custom-usage-of-xlnet) is a use case of extracting features (`seq_out` will be what you want). We might also add a python file to specifically do that in the future.
